cmd/run.go

```diff
@@ -0,0 +1,68 @@
+package cmd
+
+import (
+	"maps"
+	"slices"
+	"sync"
+	"time"
+
+	"github.com/C-L-I-M/chapter-dong-dong/config"
+	"github.com/C-L-I-M/chapter-dong-dong/discord"
+	"github.com/C-L-I-M/chapter-dong-dong/scappers"
+	"github.com/bwmarrin/discordgo"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func run(cmd *cobra.Command, _ []string) {
+	cfg, err := config.Load()
```

Comment:
`config` comes with the cobra import?

Reply:
this is the config directory in the project